test/e2e/xfn_test.go

```diff
@@ -227,3 +178,87 @@ func TestXfnRunnerImagePull(t *testing.T) {
 			Feature(),
 	)
 }
+
+func TestXfnRunnerWriteToTmp(t *testing.T) {
```

Comment:
@phisco could we have a common setup for these two tests, so that we can execute them without going through two setup and teardown rounds? e2e framework should support that through Table tests?

Reply:
talking with @ezgidemirel we saw that the two setups are slightly different. Given that we agreed to try to stick to having "1 Test, 1 feature" as much as possible, for the moment I'd keep them distinct, but we should definitely revisit it in the near future as we add more and more tests to reduce times. WDYT?